packages/expo-linear-gradient/src/NativeLinearGradient.types.ts

```diff
@@ -1,8 +1,8 @@
-import * as React from 'react';
-import { View } from 'react-native';
+import { PropsWithChildren } from 'react';
+import { ViewProps } from 'react-native';
 
-export type NativeLinearGradientProps = React.ComponentProps<typeof View> &
-  React.PropsWithChildren<{
+export type NativeLinearGradientProps = ViewProps &
+  PropsWithChildren<{
```

Comment:
If component is extended by `View`, is the `PropsWithChildren` definition still required? 

`children` prop should be inherited from the `View` anyway, right?

Reply:
Misfortunately it is not, or perhaps that's desirable behaviour as I guess some View's specifications might not accept children.
See here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/595828f2512cc4db08ec562a7a8fad078570acb5/types/react-native/index.d.ts#L2457

Reply:
@bbarthec Unfortunately the DT types for React Native are often incorrect, probably because contributing to or maintaining them is quite hard and painful.

The source of truth should be the RN Flow typing, which includes the `children`:
* https://github.com/facebook/react-native/blob/main/Libraries/Components/View/ViewPropTypes.js#L379:L388

Maybe some in near future would create PR for DT with the fix for that case. 😉